cmd/minishift/cmd/start_preflight.go

```diff
@@ -243,6 +249,20 @@ func checkKvmDriver() bool {
 	return true
 }
 
+//checkDefaultNetwork returns true if the "default" network is present
+func checkDefaultNetwork() bool {
+	cmd := exec.Command("virsh", "--connect", "qemu:///system", "net-list")
+	stdOutStdError, err := cmd.CombinedOutput()
+	if err != nil {
+		return false
+	}
+	stdOut := fmt.Sprintf("%s", stdOutStdError)
+	if strings.Contains(stdOut, "default") {
```

Comment:
can you make sure that it is only `default`, and not `mydefault` ? and also , that the network is active?

```
$ virsh --connect qemu:///system net-list
 Name                 State      Autostart     Persistent
----------------------------------------------------------
 default              active     yes           yes
 docker-machines      active     yes           yes
 vagrant-libvirt      active     no            yes
```

Reply:
i added the check for `active` but WDYT? should it be in a different function?

Reply:
haven't checked yet, but from my feeling breaking this up feels unnecessary